test/common_utils.py

```diff
@@ -388,6 +388,11 @@ def assertEqual(self, x, y, prec=None, message='', allow_inf=False):
             self.assertEqual(x.item(), y, prec, message, allow_inf)
         elif isinstance(y, torch.Tensor) and isinstance(x, Number):
             self.assertEqual(x, y.item(), prec, message, allow_inf)
+        elif isinstance(x, torch.BoolTensor) and isinstance(y, torch.BoolTensor):
```

Comment:
this looks wrong ... are BoolTensors not instances of Tensors?

Reply:
You are right, BoolTensor is an instance of Tensor and, lets say, x is 
```
tensor([[False, False],
        [False, False]], dtype=torch.bool)
```

in this case `isinstance(y, torch.Tensor)` will return true, but the rest of compare is written for a integer types. I'm getting an error: "RuntimeError: expected type torch.ByteTensor but got CPUBoolType"  on line 
`diff = a - b`